nmrfx-processor/src/main/java/org/nmrfx/processor/datasets/DatasetGroupIndex.java

```diff
@@ -14,6 +14,32 @@ public DatasetGroupIndex(int[] indices, int groupIndex) {
         this.groupIndex = groupIndex;
     }
 
+    // used from Python
+    public DatasetGroupIndex(int[] values) {
+        this.indices = new int[values.length - 1];
+        System.arraycopy(values, 0, indices, 0, indices.length);
+        this.groupIndex = values[values.length -1];
+    }
+
+    public DatasetGroupIndex(String strValue) {
+        String[] fields = strValue.split(",");
```

Comment:
Hitting Del with empty list no longer causes exception
Use * instead of -1 for all rows in dimension
Don't try to scan files with NUS lists
Format is more clear by using string descriptor for real/imaginary.  Instead of 4,7,1 use 4,7,"RR"